src/async_client.rs

```diff
@@ -260,6 +268,27 @@ impl AsyncClient {
         &self.homeserver
     }
 
+    #[doc(hidden)]
+    /// Access to the underlying `BaseClient`. Used for testing and debugging so far.
```

Comment:
No that was because the tests are in a separate folder it treats it as a new crate, I think it works just like the examples folder in this case. I just used it to print everything I can get rid of it?

Reply:
Ah right. It's not very important but it might be wise to remove it until we decide if we want to give people direct access to the base client or just expose the methods of it directly.